apps/verticalhome/js/configurator.js

```diff
@@ -188,8 +200,15 @@
     });
   }
 
+  function loadPinApps() {
+    LazyLoader.getJSON(DEFAULT_PIN_APP_CONF_FILE).then(
+      onLoadPinAppsJSON, onErrorPinApps
+    );
+  }
+
   function Configurator() {
     load();
+    loadPinApps();
```

Comment:
I'm not sure what the configurator does. Is this code that only runs the very first time after flashing the phone?  I assume that after the first run the list of pinned apps will be read from some kind of local storage, right?